routes/meeting_router.js

```diff
@@ -1,20 +1,24 @@
 const {meetingService} = require("../services/meeting_service")
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
 
-// TOMEK
-router.get('/<UUID_meeting>', function(req, res, next) {
-  res.send("Timeslots from hosts");  
+router.get('/:uuid', function(req, res, next) {
+  const uuid = req.params.uuid
+  meetingService.getTimeSlotsIntersection(uuid)
```

Comment:
Wcześniej mieliście `HostService`, a tu `meetingService` - dlaczego tak różnie pisane nazwy, czy to inne byty są?

Reply:
Wynika to pewnie z tego że nie ustaliśmy konwencji